pipelines/controller-cli/tests/test_main.py

```diff
@@ -0,0 +1,89 @@
+import contextlib
+import io
+import sys
+import unittest
+from unittest import mock
+
+from src.main import main, status
+
+
+class TestControllerCLI(unittest.TestCase):
```

Comment:
As discussed, please also update the E2E tests to use the CLI tool, e.g., [here](https://github.com/google/fhir-data-pipes/blob/046683fc26e71e8e55edf6701a0a3538c372e771/e2e-tests/controller-spark/controller_spark_sql_validation.sh#L191).